all/logging/__init__.py

```diff
@@ -65,6 +65,17 @@ def add_summary(self, name, mean, std, step="frame"):
             step (str, optional): Which step to use (e.g., "frame" or "episode")
         '''
 
+    # @abstractmethod
+    # def set_phase(self, name, mean, std, step="frame"):
+    #     # TODO: Fill
+    #     pass
+
+    @abstractmethod
+    def close(self):
```

Comment:
This abstract method was added, but not used anywhere. Should `run_experiment` call this somehow?

Reply:
This is actually called in the Experiment  `save` method. The comet writer doesn't know when to stop an experiment in its dashboard unless you run `close` or the script ends. This is an issue in jupyter notebooks running interactively because comet won't close the experiment until you start a new one or shut down jupyter. Let me know if you think this is an issue or have any better ways of handling it.

Reply:
I don't think it should be in `save` because you might want to resume the experiment after. Perhaps the right fix is to add an `experiment.close()` method.